Source/JavaScriptCore/dfg/DFGLiveCatchVariablePreservationPhase.h

```diff
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
```

Comment:
This should be automatically included by config.h.  Why are we adding these again?